tests/mock_flags_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for tests.mock_flags."""
+
+import unittest
+
+from perfkitbenchmarker import flags
+from tests import mock_flags
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('test_flag', 0, 'Test flag.')
+
+
+class MockFlagsTestCase(unittest.TestCase):
+
+  def setUp(self):
+    super(MockFlagsTestCase, self).setUp()
+    self.flags = mock_flags.MockFlags()
+
+  def testGetUnsetFlag(self):
+    self.assertIs(self.flags['test_flag'].present, False)
+
+  def testSetAndGetFlag(self):
+    self.flags.test_flag = 5
+    self.assertIs(self.flags.test_flag, 5)
+    self.assertIs(self.flags['test_flag'].present, True)
+    self.assertIs(self.flags['test_flag'].value, 5)
```

Comment:
Except for boolean and reference equality tests, I think this should be `assertEqual`